src/components/sections/Article.jsx

```diff
@@ -0,0 +1,62 @@
+import { Link, useParams } from "react-router-dom";
+import { useGetNewsQuery } from "../../features/newsSlice";
+import Wrapper from "../wrappers/Wrapper";
+
+const Article = () => {
+    const params = useParams();
+    const { articleId } = params;
+    console.log(articleId);
+    const { data, isSuccess } = useGetNewsQuery();
+    const articleData = isSuccess
+        ? data.find((article, index) => parseInt(articleId, 10) === index)
+        : null;
+    return (
+        <Wrapper>
+            {articleData !== null ? (
```

Comment:
Null is falsy so you don't need to explicitly compare it
```suggestion
            {articleData ? (
```

Reply:
got it, thanks